native_client/java/libdeepspeech/src/androidTest/java/org/mozilla/deepspeech/libdeepspeech/test/BasicTest.java

```diff
@@ -149,4 +151,33 @@ public void loadDeepSpeech_sttWithMetadata_withLM() {
         assertEquals("she had your dark suit in greasy wash water all year", decoded);
         m.freeModel();
     }
+
+    @Test
+    public void loadDeepSpeech_HotWord_withLM() {
+        DeepSpeechModel m = new DeepSpeechModel(modelFile);
+        m.enableExternalScorer(scorerFile);
+
+        for(int i = 0; i < word.length; i++) {
+            m.addHotWord(word[i], boost[i]);
+            String decoded = doSTT(m, false);
+            assertEquals("she had your dark suit in greasy wash water all year", decoded);
+            m.eraseHotWord(word[i]);
+        }
+
+        m.freeModel();
+    }
+
+    @Test
+    public void loadDeepSpeech_HotWord_noLM() {
+        DeepSpeechModel m = new DeepSpeechModel(modelFile);
+
+        for(int i = 0; i < word.length; i++) {
```

Comment:
no need for the for loop

Reply:
without for loop how can we check for different boost values ? @lissyx

Reply:
we don't care here, we just want to check if `addHotWord` called without an external scorer properly throws an exception